### PR TITLE
tauqeer adding html project to new repo

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
Added html project to new repo

### CODE REVIEW

**Review by Tanvir04khan — APPROVED** [review]

File: tasks-by-users/Tauqeer-Ahmed-99/First Webpage/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html>
+    <head>
+        
+        <!-- 
+        Created & Owned by Tauqeer Khan
+        Date:16/07/21
+        -->
```
Comment (faran4engg) [inline tasks-by-users/Tauqeer-Ahmed-99/First Webpage/index.html:8]: also maintain it with prettier

Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/First Webpage/index.html]: done with prettier changes... @faran4engg

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css [hunk 2]
```diff
@@ -3,235 +3,212 @@ Created, maintained & Owned by: Tauqeer Khan
...
 
-body{
-    font-family: arial;
-    margin: 10px auto;
-    background-color: lightgray;
-    
+body {
+  font-family: arial;
+  margin: 10px auto;
+  background-color: lightgray;
```
Comment (coder-mezab) [inline tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css:9]: you can just go with `background` here totally up to you

Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css:9]: yep, I will consider these changes, these projects were coded when we were learning CSS and were not good at it. @coder-mezab

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css [hunk 3]
```diff
@@ -3,235 +3,212 @@ Created, maintained & Owned by: Tauqeer Khan
...
-    width: 300px;
-    margin: 10px auto;
-    padding: 20px auto;
-    
+.span-list a img {
+  display: block;
+  height: 140px;
+  width: 300px;
+  margin: 10px auto;
+  padding: 20px auto;
```
Comment (coder-mezab) [inline tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css:88]: again grp the similar properties @Tauqeer-Ahmed-99

Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css:88]: yep, I will consider these changes, these projects were coded when we were learning CSS and were not good at it. @coder-mezab

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css [hunk 4]
```diff
@@ -3,235 +3,212 @@ Created, maintained & Owned by: Tauqeer Khan
...
-    margin: 0px;
-    border: 2px black dashed;
-    text-align: center;
-    color: black
+#customer {
+  text-decoration: none;
+  font-family: arial;
+  display: block;
+  width: 180px;
+  margin: 0px;
```
[the remark this answers is not in the document]
Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css:199]: yep, I will consider these changes, these projects were coded when we were learning CSS and were not good at it. @coder-mezab

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css [hunk 5]
```diff
@@ -3,235 +3,212 @@ Created, maintained & Owned by: Tauqeer Khan
...
 }
-#contact{
-     width: 1000px;
-    margin: 0 auto;
-    background: inherited;    
-
+#contact {
+  width: 1000px;
+  margin: 0 auto;
+  background: inherited;
```
[the remark this answers is not in the document]
Comment (faran4hatch) [reply tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css:150]: this should be corrected. The rest of the comments we can ignore but this is the value of the property. We cannot ignore it.

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (faran4hatch) [reply tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css:199]: I agree... but this time travel activity is good for all of us to see how far we came from being noob to a professional developer who is concerned about good practices :)

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css]: yes I changed it but forgot to push now I pushed the code you can check now. @faran4engg

**Comment by Tauqeer-Ahmed-99** [comment]
> the folder should br `01-first-webpage` @Tauqeer-Ahmed-99

Done this change

**Comment by fayez-baig** [comment]
@Tauqeer-Ahmed-99 please add @NextGenDevOrg/all-reviewers  in the reviewers section

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/Contact-us.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  <head>
+    <title>Contact US and Support</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css" />
+  </head>
+  <body id="contact">
+    <div id="contact-us-page">
+      <h1>Contact US and Support</h1>
+      <p>Address.:601, Uzair Tower, Next to Zoya Complex, ********.</p>
+      <p>Mob.No.:+91 887999****</p>
+      <p>E-mail.:tauqueerrkhan@gmail.com</p>
```
Comment (fayez-baig) [inline tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/Contact-us.html:12]: try  using tel: inside the href attribute @Tauqeer-Ahmed-99 
 
try using mailto: inside the href attribute @Tauqeer-Ahmed-99

**Review by fayez-baig — APPROVED** [review]

File: tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/Contact-us.html (hunk 6 above)
Comment (Tauqeer-Ahmed-99) [reply tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/Contact-us.html]: done changes @fayez-baig

**Comment by Tauqeer-Ahmed-99** [comment]
> 33+ comments. Inspired by @fayez-baig 😂

l😂😂😂😂

**Comment by Tauqeer-Ahmed-99** [comment]
> 33+ comments. Inspired by @fayez-baig 😂

I am still unable to merge @faran4engg

**Review by faran4engg — APPROVED** [review]

**Pull request merged**